packages/core/src/Asset/index.ts

```diff
@@ -1,2 +1,3 @@
 export * as util from './util';
 export * from './types';
+export * as cip67 from './cip67';
```

Comment:
I take back my previous suggestion about the namespace, I think it's too nested when importing from '@cardano-sdk/core': `Asset.cip67.AssetNameLabel.decode` -> `Asset.AssetNameLabel.decode`
```suggestion
export * from './cip67';
```

Reply:
Fixed: https://github.com/input-output-hk/cardano-js-sdk/pull/767/commits/1032606789f812ae16dc643c27e091c58a8d0c09